gulpfile.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+
+var browserify = require('browserify');
+var gulp = require('gulp');
+var source = require('vinyl-source-stream');
+var buffer = require('vinyl-buffer');
+var uglify = require('gulp-uglify');
+var sourcemaps = require('gulp-sourcemaps');
+var gutil = require('gulp-util');
+var watchify = require('watchify');
+var child = require('child_process');
+var http = require('http');
+var ecstatic = require('ecstatic');
+var path = require('path');
+
+var settings = {
+  cache: {},
+  packageCache: {},
+  debug: true,
+  watch: false,
+  compression: null
+};
+
```

Comment:
> Are these settings good for a release build?

Yes, but that's an excellent question, I'll add comments to explain what those prefs are for.